jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/feedback/BackdropTests.java

```diff
@@ -1,22 +1,23 @@
 package io.github.epam.material.tests.feedback;
 
+import com.epam.jdi.tools.Timer;
 import io.github.epam.TestsInit;
 import org.testng.annotations.Test;
-
-import static io.github.com.StaticSite.backdropFrame;
-import static io.github.com.StaticSite.feedbackBackdropDefaultPage;
+import static io.github.com.MaterialNavigator.openSection;
+import static io.github.com.pages.feedback.BackdropPage.*;
 
 public class BackdropTests extends TestsInit {
 
+    private static Timer timer = new Timer(1000L);
+
     @Test
     public void defaultBackdropTest() {
-        feedbackBackdropDefaultPage.open();
+        openSection("Backdrop");
 
-        backdropFrame.backdrop.is().hidden();
-        backdropFrame.showBackdropButton.click();
-        backdropFrame.backdrop.is().visible();
-        backdropFrame.backdrop.hide();
-        backdropFrame.backdrop.is().hidden();
+        timer.wait(() -> backdrop.is().hidden());
+        showBackdropButton.click();
+        backdrop.is().visible();
+        backdrop.hide();
+        timer.wait(() -> backdrop.is().hidden());
     }
-
```

Comment:
Is it possible to delete Backdrop.class and BackdropAssert.class? Possible use UIElement instead of Backdrop in BackdropPage